microsetta_interface/templates/new_participant.jinja2

```diff
@@ -243,18 +258,24 @@
     </div>
     <div class="row mb-4">
         {% if language_tag != "ja_JP" %}
-        <div class="col-md">
-            <input type="radio" id="age-range-1" name="age-range" value="0-6" onchange="change_consent_div(this)">
-            <label for="age-range-1" class="consent-age-label">{{ tl['PARTICIPANT_FORM']['AGE_0_6'] |e}}</label>
-        </div>
-        <div class="col-md">
-            <input type="radio" id="age-range-2" name="age-range" value="7-12" onchange="change_consent_div(this)">
-            <label for="age-range-2" class="consent-age-label">{{ tl['PARTICIPANT_FORM']['AGE_7_12'] |e}}</label>
-        </div>
-        <div class="col-md">
-            <input type="radio" id="age-range-3" name="age-range" value="13-17" onchange="change_consent_div(this)">
-            <label for="age-range-3" class="consent-age-label">{{ tl['PARTICIPANT_FORM']['AGE_13_17'] |e}}</label>
-        </div>
+            {% if cur_age_index <= 0 %}
```

Comment:
This and below is testing against a magic number. Can this be changed to test against an exact value, eg `if cur_age in ('UNKNOWN', '0-6')`? Though ideally these aren't strings but instead string literals though that may be weird crossing the boundary into jinja

Reply:
Changed the magic number situation. The age ranges are now string literals in implementation.py, but I agree that copying them into the jinja2 template would be both weird and clunky. Other than that, this PR is ready for re-review, but it won't pass integration tests until the corresponding Private API PR has been merged.